prow/cluster/cert-manager-deployment.yaml

```diff
@@ -0,0 +1,259 @@
+---
```

Comment:
Looks like this is more than just the deployment. So can you rename this to just cert-manager.yaml?

Reply:
(Follow the getting_started and grandmatriarch patterns)

Reply:
yeah got it, there's `MULTI_KIND` :-)